### PR TITLE
Reset evals 397

### PR DESCRIPTION
**Pull request by solijonovam**
What we did:

-Added a 'reset' button, and a modal that pops up once the 'reset' button is clicked
-The modal confirms resetting the form
-once confirmed, the form will be deleted
-redirects to the original form

To test:
-Evaluate a student
-Click 'show evaluation' 
-once in the form, 'reset' button will be there. 
-reset the form
-run the test suite

### CODE REVIEW

**Comment by tylerpar99** [comment]
This PR isn't ready yet still working on the testing suite.

File: app/controllers/main_routes/studentLaborEvaluation.py [hunk 1]
```diff
@@ -124,11 +126,23 @@ def sle(statusKey):
...
+    print("we re here")
+    print(sleForm.validate_on_submit())
+    # print(sleForm.attendance.data)
+    # print(sleForm.accountability.data)
+    # print(sleForm.teamwork.data)
+    # print(sleForm.initiative.data)
+    # print(sleForm.respect.data)
+    # print(sleForm.learning.data)
+    # print(sleForm.jobSpecific.data)
+    # print(sleForm.isSubmitted)
```
Comment (gahimbaref) [inline app/controllers/main_routes/studentLaborEvaluation.py]: Forgot to remove debug print statements

File: app/controllers/main_routes/studentLaborEvaluation.py [hunk 2]
```diff
@@ -124,11 +126,23 @@ def sle(statusKey):
...
+    print(request.form)
     if sleForm.validate_on_submit():
+        print("here?")
```
Comment (gahimbaref) [inline app/controllers/main_routes/studentLaborEvaluation.py]: remove print statement

File: app/controllers/main_routes/studentLaborEvaluation.py (hunk 1 above)
Comment (tylerpar99) [reply app/controllers/main_routes/studentLaborEvaluation.py]: This PR is still being worked on

**Review by sreynit02 — CHANGES_REQUESTED** [review]
I suggest that the modal should have a bit more explanation of what will be erased once they successfully reset.

**Review by sreynit02 — CHANGES_REQUESTED** [review]
image
The modal looks empty, I suggest adding a body to the modal say sth along this line. "This action will erase comments in all section."

**Comment by tylerpar99** [comment]
> image The modal looks empty, I suggest adding a body to the modal say sth along this line. "This action will erase comments in all section."

Fixed

**Review by sreynit02 — APPROVED** [review]
Great job!

**Pull request merged**